spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -131,6 +131,48 @@ configuration.classes=org.springframework.cloud.stream.app.file.sink.FileSinkPro
 configuration-properties.names=server.port
 ```
 
+[IMPORTANT]
+====
+As of Spring Cloud Data Flow `1.0.0.RELEASE` the whitelisting of application properties
+is only explicitly supported for Spring Boot `1.3.x` based application. Milestone releases
+of the upcoming `1.4.0` release are not explicitly supported, yet.
+
+The cause is in the used `spring-boot-maven-plugin`. As a result you will observe
+all available application properties being listed, not just the whitelisted ones. As a temporary
```

Comment:
We need documentation that talks about `dataflow` server (enabled using @EnableDataflowServer) that uses boot version `1.4.x` as well.